docs/source/common/lightning_module.rst

```diff
@@ -54,7 +55,7 @@ Notice a few things.
         new_x = torch.Tensor(2, 3)
         new_x = new_x.type_as(x)
 
-5.  Lightning by default handles the distributed sampler for you.
+5.  Lightning, by default, handles the distributed sampler for you when running under a distributed strategy.
```

Comment:
I liked the "for you" statement more

Reply:
just trying to be a little more explicit about when actually lightning will handle distributed sampler.